src/xenia/cpu/xex_module.h

```diff
@@ -186,6 +200,9 @@ class XexModule : public xe::cpu::Module {
   uint32_t base_address_ = 0;
   uint32_t low_address_ = 0;
   uint32_t high_address_ = 0;
+
+  XexFormat xex_format_;
```

Comment:
You've left this uninitialized here, I think the best case here would be to have another enum (`kFormatXexUnknown`) to initialixe `xex_format_` to.